README.md

```diff
@@ -250,3 +250,20 @@ If you need to read from the state to change it, you should do this inside the a
 ## Licenses and Attributions
 
 -   License information about used images can be found [here](frontend/src/assets/image-sources.md). All images are licensed under their original license.
+
+## Contributors
```

Comment:
@christianzoellner @mbarkowsky Do you want to be mentioned here?

Reply:
I would mention them here. The emoji keys (see comment below) clarify their role in the project.